client/css/style.css

```diff
@@ -21,19 +23,20 @@ section {
     width: 100%;
     padding: 20px;
 }
-
-#messages {
-    list-style-type: none;
-    margin: 0;
-    padding: 0;
-}
-
-#messages li {
-    padding: 5px 10px;
+#message{
+    position:absolute;
```

Comment:
Use `position: fixed` Stronger certainty that it stays in the middle of the screen regardless of other conditions